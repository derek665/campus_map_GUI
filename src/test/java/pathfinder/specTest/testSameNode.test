# test correct output when the start node and end node are the same

CreateGraph testSameNode

AddNode testSameNode A

FindPath testSameNode A A