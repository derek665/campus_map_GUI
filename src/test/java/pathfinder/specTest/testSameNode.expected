# test correct output when the start node and end node are the same

created graph testSameNode

added node A to testSameNode

path from A to A:
total cost: 0.000